.gitignore

```diff
@@ -221,5 +221,3 @@ fabric.properties
 unit_test_coverage/
 test-results.xml
 .env.*
```

Comment:
Waarom moet requirements uit de gitignore?

Reply:
Omdat je deze wel wilt meeleveren voor snelle checkout/opzet van een repository. Het is dan de taak van de developer/committer om te zorgen dat de requirements.txt files beschikbaar zijn,  en niet aan de gebruiker

Het voorkomt ook dat de gebruiker pip-tools moet installeren voordat hij/zij het repository werkend kan krijgen

Reply:
Prima dan laten we het zo